hack/check-skaffold-deps.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+operation="${1:-check}"
+
+echo "> ${operation^} Skaffold Dependencies"
+
+success=true
+repo_root="$(git rev-parse --show-toplevel)"
+
+function run() {
+  if ! "$repo_root"/hack/check-skaffold-deps-for-binary.sh "$operation" --skaffold-file "$1" --binary "$2" --skaffold-config "$3"; then
```

Comment:
Isn't it possible to re-use the original script from Gardener similarly to https://github.com/gardener/gardener-discovery-server/blob/main/hack/check-skaffold-deps.sh ?

Reply:
Yeah, nice idea, thanks : )